rootfs/api/views.py

```diff
@@ -38,6 +38,7 @@ def get(self, request):
                 c.execute("SELECT 0")
         except django.db.Error as e:
             logger.critical("Database health check failed")
+            # FIXME why is turning on DEBUG=true in env not outputting this error?
```

Comment:
I made https://github.com/deis/controller/issues/568 for this. Had to change it to `critical` in my local tests